core/src/com/po/fuck/model/constants/BordersConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.po.fuck.model.constants;
+
+import java.nio.file.Paths;
+import java.util.Properties;
+import static com.po.fuck.model.constants.ConstantsLoader.loadProperties;
+import static com.po.fuck.model.constants.ConstantsLoader.loadInt;
+
+
+public final class BordersConstants extends BaseConstants {
```

Comment:
Do they need their own constants ?

Reply:
Hmm. Can move it to `Position` ?

Reply:
Sounds not very great, because then we need to do something with speed.

Reply:
?

Reply:
If we have position constants, then the speed should be there also, because it also depends on a lengths/etc.

Reply:
Can we simply remove borders ?